persistence/schema/gov.go

```diff
@@ -1,442 +1,145 @@
-		hex.EncodeToString(params.ServiceNodeMaxChainsOwner),
-		hex.EncodeToString(params.ServiceNodeUnstakingBlocksOwner),
-		hex.EncodeToString(params.ServiceNodeMinimumPauseBlocksOwner),
-		hex.EncodeToString(params.ServiceNodeMaxPausedBlocksOwner),
-		hex.EncodeToString(params.ServiceNodesPerSessionOwner),
-		hex.EncodeToString(params.FishermanMinimumStakeOwner),
-		hex.EncodeToString(params.FishermanMaxChainsOwner),
-		hex.EncodeToString(params.FishermanUnstakingBlocksOwner),
-		hex.EncodeToString(params.FishermanMinimumPauseBlocksOwner),
-		hex.EncodeToString(params.FishermanMaxPausedBlocksOwner),
-		hex.EncodeToString(params.ValidatorMinimumStakeOwner),
-		hex.EncodeToString(params.ValidatorUnstakingBlocksOwner),
-		hex.EncodeToString(params.ValidatorMinimumPauseBlocksOwner),
-		hex.EncodeToString(params.ValidatorMaxPausedBlocksOwner),
-		hex.EncodeToString(params.ValidatorMaximumMissedBlocksOwner),
-		hex.EncodeToString(params.ValidatorMaxEvidenceAgeInBlocksOwner),
-		hex.EncodeToString(params.ProposerPercentageOfFeesOwner),
-		hex.EncodeToString(params.MissedBlocksBurnPercentageOwner),
-		hex.EncodeToString(params.DoubleSignBurnPercentageOwner),
+	fields := "name,height,value,enabled,type"
 
-		hex.EncodeToString(params.MessageDoubleSignFeeOwner),
-		hex.EncodeToString(params.MessageSendFeeOwner),
-		hex.EncodeToString(params.MessageStakeFishermanFeeOwner),
-		hex.EncodeToString(params.MessageEditStakeFishermanFeeOwner),
-		hex.EncodeToString(params.MessageUnstakeFishermanFeeOwner),
-		hex.EncodeToString(params.MessagePauseFishermanFeeOwner),
-		hex.EncodeToString(params.MessageUnpauseFishermanFeeOwner),
-		hex.EncodeToString(params.MessageFishermanPauseServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageTestScoreFeeOwner),
-		hex.EncodeToString(params.MessageProveTestScoreFeeOwner),
-		hex.EncodeToString(params.MessageStakeAppFeeOwner),
-		hex.EncodeToString(params.MessageEditStakeAppFeeOwner),
-		hex.EncodeToString(params.MessageUnstakeAppFeeOwner),
-		hex.EncodeToString(params.MessagePauseAppFeeOwner),
-		hex.EncodeToString(params.MessageUnpauseAppFeeOwner),
-		hex.EncodeToString(params.MessageStakeValidatorFeeOwner),
-		hex.EncodeToString(params.MessageEditStakeValidatorFeeOwner),
-		hex.EncodeToString(params.MessageUnstakeValidatorFeeOwner),
-		hex.EncodeToString(params.MessagePauseValidatorFeeOwner),
-		hex.EncodeToString(params.MessageUnpauseValidatorFeeOwner),
-		hex.EncodeToString(params.MessageStakeServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageEditStakeServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageUnstakeServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessagePauseServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageUnpauseServiceNodeFeeOwner),
-		hex.EncodeToString(params.MessageChangeParameterFeeOwner),
-		DefaultBigInt,
-	)
-}
+	subQuery := fmt.Sprintf(`SELECT %s`, fields)
+	//TODO (@deblasis): clarify if and how we should use `enabled` here
```

Comment:
I think I need to better understand your train of thought around the enabled flag. Not sure if this is correct,